Card/src/main/java/com/braintreepayments/api/CardAnalytics.java

```diff
@@ -0,0 +1,20 @@
+package com.braintreepayments.api;
+
+enum CardAnalytics {
+
+    // Tokenize Events
+    CARD_TOKENIZE_STARTED("card:tokenize:started"),
+    CARD_TOKENIZE_FAILED("card:tokenize:failed"),
+    CARD_TOKENIZE_SUCCEEDED("card:tokenize:succeeded"),
+    CARD_TOKENIZE_NETWORK_CONNECTION_LOST("card:tokenize:network-connection:failed");
```

Comment:
I think we actually got rid of this one on BT before GA! Unless there is a specific reason that we want it on Android.

Reply:
Fixed!